editor/plugins/org.fusesource.ide.wsdl2rest/pom.xml

```diff
@@ -0,0 +1,237 @@
+            <artifactId>wsdl4j</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>commons-lang</groupId>
+            <artifactId>commons-lang</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>commons-validator</groupId>
+            <artifactId>commons-validator</artifactId>
+        </dependency>
+    </dependencies>
+  
+    <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-dependency-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>copy-wsdl2rest</id>
+            <phase>generate-sources</phase>
+            <goals>
+              <goal>copy</goal>
+            </goals>
+            <configuration>
+              <stripVersion>true</stripVersion>
+              <outputDirectory>lib</outputDirectory>
+              <artifactItems>
+                <!-- Impl -->
+                <artifactItem>
+                  <groupId>org.jboss.fuse.wsdl2rest</groupId>
+                  <artifactId>wsdl2rest-impl</artifactId>
+                  <version>${version.wsdl2rest}</version>
+                </artifactItem>
+                <!-- Util -->
+                <artifactItem>
+                  <groupId>org.jboss.fuse.wsdl2rest</groupId>
+                  <artifactId>wsdl2rest-util</artifactId>
+                  <version>${version.wsdl2rest}</version>
+                </artifactItem>
+               </artifactItems>
+            </configuration>
+          </execution>
+          <execution>
+            <id>copy-dependencies</id>
+            <phase>generate-sources</phase>
+            <goals>
+              <goal>copy-dependencies</goal>
+            </goals>
+            <configuration>
+              <excludeScope>system</excludeScope>
+              <outputDirectory>lib</outputDirectory>
+            </configuration>
+          </execution>
+        </executions>
+      </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-clean-plugin</artifactId>
+        <configuration>
+          <filesets>
+            <fileset>
+              <directory>lib</directory>
+              <includes>
+                <include>**/*</include>
+              </includes>
+            </fileset>
+          </filesets>
+        </configuration>
+      </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-enforcer-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>ban-uberjars</id>
+            <configuration>
+              <skip>true</skip>
+            </configuration>
+          </execution>
+        </executions>
+      </plugin>
+    </plugins>
+  </build>
+  
+  <repositories>
+    	<repository>
+  		<id>Saved staging repo for cxf</id>
+  		<url>https://repository.jboss.org/nexus/content/repositories/fusesource_releases_external-9109/</url>
```

Comment:
comment to not forget to be sure that all the repository are saved before we do a merge in master